main.tf

```diff
@@ -175,8 +175,9 @@ module "graphdb" {
 
   # VM Image
 
-  ami_id          = var.ami_id
-  graphdb_version = var.graphdb_version
+  ami_id              = var.ami_id
+  graphdb_version     = var.graphdb_version
+  aws_custom_owner_id = var.aws_custom_owner_id
```

Comment:
I don't really like the naming here, I suggest we rename this to override_owner_id or something like this.

Reply:
Got it. It was fixed.